src/main/datalog_console/chrome/extension/background/main.cljs

```diff
@@ -0,0 +1,67 @@
+(ns datalog-console.chrome.extension.background.main
+  (:require [goog.object :as gobj]))
+
+
+(defonce tools-conns* (atom {}))
+(defonce remote-conns* (atom {}))
+
+
+(defn handle-devtool-message [devtool-port message _port]
+  (let [tab-id (gobj/get message "tab-id")]
+    (cond
+      (= "init" (gobj/get message "name"))
```

Comment:
“name” seems like it could be ambiguous. Is this scoped in such a way that messages could only come from datalog-console or should this be fully qualified?

Reply:
"name" seems to be required. https://developer.chrome.com/docs/extensions/reference/runtime/#method-connect